core-bundle/src/Resources/contao/library/Contao/Picture.php

```diff
@@ -103,9 +103,15 @@ public static function create($file, $size=null)
 			$size = (int) $size[2];
 		}
 
+		// predefined image size as resize mode
+		elseif (\is_array($size) && !empty($size[2]) && is_string($size[2]) && $size[2][0] === '_')
```

Comment:
The same here.

Reply:
Done in e97bba284687fa9c6db236f032eebf2238922d6a